### ISSUE TITLE
 [Guided tours] start url validation

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#40020]
### Steps to reproduce the issue
When creating a tour you are asked to insert a url for the start location. This must be a relative URL. There is no validation to ensure that you enter a relative url. If you dont enter a relative url then the tour will fail (obviously) and the only option is to close the browser to kill the session.
 
Creating seperate issue for each bug report so that they dont get lost inside a bigger report (again)

**Comment by obuisard** [comment]
Agreed, we should have some kind of validation, even though the inline help gives specific indication.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40020">issues.joomla.org/tracker/joomla-cms/40020</a>.</sub>

**Comment by brianteeman** [comment]
Closing as it has a dedicated issue on the guided tours project https://github.com/joomla-projects/guided-tours/issues/29 - no point in it existing in two places.
